### ISSUE TITLE
Download Troubleshoot output

### ISSUE THREAD

**Issue by Sanaz01** [FredHutch/shiny-cromwell#94]
Can there be a feature to download the output of "Troubleshoot a Workflow" in Troubleshoot tab?

**Comment by sckott** [comment]
see also #7 (b/c both are about downloading details of that page/section)

**Comment by sckott** [comment]
Should be easy enough, though I think we'd want to add this as we change interfaces across the app to download outputs.

**Comment by tefirman** [comment]
Still think this is medium priority, but I wonder if implementing this will become easier after the GUI update associated with v1.2 (i.e. as a visualization rather than a download), so I'm setting this as v1.3 for now.

**Comment by sckott** [comment]
Yeah, I think there's an opportunity to help the user troubleshoot in the browser, let's see what we can do
